invisible_cities/cities/components.py

```diff
@@ -589,6 +595,31 @@ def MC_hits_from_files(files_in : List[str], rate: float) -> Generator:
                        timestamp    = timestamp(evt))
 
 
+def dhits_from_files(paths: List[str]) -> Iterator[Dict[str,Union[HitCollection, pd.DataFrame, MCInfo, int, float]]]:
```

Comment:
It seems to me that 
`hits_and_kdst_from_files`, `cdst_from_files` and now a newly added `dhits_from_files` are all doing almost the same thing! 
They all read hits (from different tables) and optionally `kdst` and `summary_df`, but do we really need 3 different readers? Cant we just have one reader with hits- table and group name as parameters and optionally read `kdst` and `summary_df`? @gonzaponte , @Aretno

Reply:
Yes, all that needs some refactoring, but I would put it in another PR.
There is `dst_from_files` which is almost what you request, but there are some subtleties done in some of those readers that might not be easy to implement in a general reader.